src/Reenbit.ChuckNorris.DataAccess/Mapping/JokeMap.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Reenbit.ChuckNorris.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Reenbit.ChuckNorris.DataAccess.Mapping
+{
+    public class JokeMap : IEntityTypeConfiguration<Joke>
```

Comment:
internal, this class will be used only inside the current library

Reply:
+